site/docs/master/output-file-format.md

```diff
@@ -97,3 +97,16 @@ resources/
                 ...
     ...
 ```
+
+## Output File Format Versioning
+
+The Velero output file format is intended to be relatively stable, but may change over time in order to support new features.
+
+In order to accommodate this, Velero follows [Semantic Versioning](http://semver.org/) for the file format version.
+
+Minor and patch versions will indicate backwards-compatible changes that previous versions of Velero can restore, including new directories or files.
+
+A major version would indicate that a version Velero older than the version that created the backup could restore it, usually because of moved or renamed directories or files.
+
+Major versions of the file format are tied to major versions of Velero, though they are not necessarily in sync - Velero 3.0 may still use backup file format 2.0, as an example.
```

Comment:
This line is a little confusing - I *think* what you're trying to convey is:
- we won't bump the major version of the backup file format outside of a velero major version bump
- a Velero major version bump doesn't necessarily mean the file format gets a major version bump

is that right?

Reply:
Yeah, that's correct. I'll try to clarify this.

Reply:
Let me know if the new changes make this more clear.